src/main/java/io/github/nucleuspowered/nucleus/modules/ticket/handlers/TicketHandler.java

```diff
@@ -0,0 +1,292 @@
+/*
+ * This file is part of Nucleus, licensed under the MIT License (MIT). See the LICENSE.txt file
+ * at the root of this project for more details.
+ */
+package io.github.nucleuspowered.nucleus.modules.ticket.handlers;
+
+import com.github.benmanes.caffeine.cache.AsyncLoadingCache;
+import com.github.benmanes.caffeine.cache.Caffeine;
+import com.github.benmanes.caffeine.cache.RemovalListener;
```

Comment:
From what I can see it is still part of [Sponge's dependencies](https://github.com/SpongePowered/SpongeCommon/blob/bleeding/gradle/implementation.gradle#L125), which I suppose solves this issue as I wouldn't need to bundle it.